docs/src/main/asciidoc/_javaee-guide/Jakarta_Mail_API.adoc

```diff
@@ -0,0 +1,4 @@
+[[Jakarta_Mail_API]]
```

Comment:
Not by this change, but all the blocks Ids which are using "(" and ")" characters make the documentation render badly. See https://docs.wildfly.org/22/JavaEE_Tutorial.html#JavaMail_API for example.


@RanabirChakraborty Would you mind removing these invalid characters to make the documentation render correctly? It is not really on the scope of this Jira, but it sounds appropriate to use this one.

I'm pointing you to the blocks IDs, ror example, replace `[[Java_Authorization_Contract_for_Containers_(JACC)]]` by `[[Java_Authorization_Contract_for_Containers_JACC]]` in Java_Authorization_Contract_for_Containers_(JACC).adoc file and so on.

Reply:
I have done the same changes here - https://github.com/wildfly/wildfly/pull/13953/files , please take a look :) I'll take care of others too.

Reply:
I could be missing something but what I can see is you are changing the file names, but not the block IDs.

The file names can contain "(" and ")" characters, but the Block IDs, those are the headers with "[[" and "]]" markers cannot contain those characters, for example, Java_Authorization_Contract_for_Containers_(JACC).adoc, line 1, contains `[[Java_Authorization_Contract_for_Containers_(JACC)]]` but it should be `[[Java_Authorization_Contract_for_Containers_JACC]]`

Reply:
Thanks! got it now :) 
https://github.com/wildfly/wildfly/pull/13953/files#diff-26d164d098fc71f48b3264385036a769956734ec202f9a1fe7ae1f3c8df090a7
did it for other specifications, forgot to do for JACC.

Reply:
Ok @RanabirChakraborty , it is difficult to track all of these changes across all the PRs. I understand your idea is to change all of these wrong headers on each PR, right? If so, take into account that the JSF one was not fixed, see https://github.com/wildfly/wildfly/pull/13935/files#diff-e3b6f2a3adda324b932ed5156478991589b9656814257e8b1891a4fc9ffe1657R1

In any case, I think the easier and quicker solution could be to review all of these at the end when WFLY-14061 is resolved to verify we haven't missed any.

Reply:
Yeah @yersan sorry that was my bad. JSF was the first one to merge and at that time I haven't got many ideas bout the whole thing. I have it in my mind that I need to do a few more changes for JSF. I'll do that.
Yes, it's also a really good idea to review all of these after all the sub-tasks are merged. I'll send a separate PR with all the leftover changes and link the same to the main issue - https://issues.redhat.com/browse/WFLY-14061 and PR - https://github.com/wildfly/wildfly/pull/13808